### ISSUE TITLE
MSVC Compiler strikes to compile

### ISSUE THREAD

**Issue by paule32** [doxygen/doxygen#11050]
I am not able to compile for Microsoft Windows:
[Windows MSVC Action](https://github.com/paule32/doxygen/actions/runs/10234653636/job/28314411220)

I can not locate the Problem.
Thanks for a helping Hand.

**Comment by albert-github** [comment]
- From which PR does this Action come?

**Comment by paule32** [comment]
it is sync forked of Doxygen - no changes on Workflow Files.
But changes in some .cpp Files, because std::min and std::max will handled different by GCC, CLANG, and MSVC

**Comment by albert-github** [comment]
See the comments posted with the relevant PR, this does not look like a problem introduced in the changes there.

**Comment by doxygen** [comment]
Missing reproduction info. Please follow the template when filing issues.

**Comment by paule32** [comment]
The Problem is, Doxygen for Linux will be built, but not for Windows, with the MSVC:
![grafik](https://github.com/user-attachments/assets/f65dcad3-b269-4864-affd-7039203e401e)

**Comment by paule32** [comment]
Hello,
I change to the Original Code in src/main.cpp and remove the Windows Stuff.
All what I changed was the thing with the std::min, and std::max.
Because, I went into trouble, when I let the Run to compile with MSVC.
So, there is no Code anymore, that is pure/native Windows API Code in the source tree.
Before my changes, I added __GNUC__ stuff, I had problem with MSVC, too.
So, something magic goes on, when compiling with CLANG or GNU C/C++ (which worked)
and something magic goes on, when compiling with MSVC.
So, my dirty, and fast Test's was the thing with "using namespace std;".

Either, std::min, and std::max was not accepted by MSVC - like I already wrote: a missing iostream header - or using namespace std.
I have no problems to compile for/with CLANG, and GNUC.

So, I ask again:
Does the workfilow Files run on my Account Scope - or is the Scopy of doxygen/doxygen involved ?
So, I have to rename the Fork, and doing commits to a Copy, that is not offically under the original Repro ?

But, how can I stop/cancel a Run ?
Because the Workflow hangs in the MSVC Debug stage, and I can not see, how to Cancel or delete this Run.
